test/proxy/utils/Initializable.test.js

```diff
@@ -213,7 +213,7 @@ contract('Initializable', function () {
     it('old and new patterns in good sequence', async function () {
       const ok = await DisableOk.new();
       await expectEvent.inConstruction(ok, 'Initialized', { version: '1' });
-      await expectEvent.inConstruction(ok, 'Initialized', { version: '255' });
+      await expectEvent.inConstruction(ok, 'Initialized', { version: '18446744073709551615' }); // MAX_UINT64
```

Comment:
Would the test support an hex syntax ? IMO `0xFFFFFFFFFFFFFFFF` would be more readable

Reply:
Unfortunately not, but we can do `BigInt('0xFFFFFFFFFFFFFFFF').toString()` which is similar imo. What do you think?

Reply:
`(2n ** 64n - 1n).toString()` might be slightly better.

Reply:
I like this one better. Updated.